ncloud/resource_ncloud_public_ip.go

```diff
@@ -167,9 +170,20 @@ func resourceNcloudPublicIpUpdate(d *schema.ResourceData, meta interface{}) erro
 		}
 
 		if len(n.(string)) > 0 {
-			if err := associatedPublicIp(d, config); err != nil {
+			if err := resource.Retry(d.Timeout(schema.TimeoutDelete)-time.Minute*19, func() *resource.RetryError {
```

Comment:
```suggestion
			if err := resource.Retry(time.Minute, func() *resource.RetryError {
```

1분 timeout 설정이시면 요렇게 해주시면 될것 같습니다~!

Reply:
네 수정했습니다~